spec/features/recipes/total_cost_spec.rb

```diff
@@ -0,0 +1,33 @@
+# User Story 3 - Total Cost
```

Comment:
This test should not be in a file called `total_cost_spec` .... the feature that we are testing is for the recipes show page, and therefore this test suite should be in there.